contractsV5/KyberNetwork.sol

```diff
@@ -675,10 +675,86 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
         }
     }
     
+    event HandleFees(address indexed recipient, uint fees);
```

Comment:
should be named
handlePlatformFee

Reply:
should be handlePlatformFee or HandlePlatformFees?